doc/Language/5to6-overview.pod6

```diff
@@ -5,8 +5,8 @@
 =SUBTITLE How do I do what I used to do?
 
 These documents should not be mistaken for a beginner tutorial or a
-promotional overview of Raku; it is intended as a technical reference
-for Raku learners with a strong Perl 5 background and for anyone
+promotional overview of Raku (née Perl 6); it is intended as a technical
```

Comment:
I don't know what's “née” is. I guess there are other readers like me.

Reply:
"Née" is a very good word to use in this situation :grin:

Reply:
“formerly Perl 6”? Easier to understand.